### ISSUE TITLE
FzfLua marks jumps to the line, not the position

### ISSUE THREAD

**Issue by damianoognissanti** [ibhagwan/fzf-lua#890]
When using the marks filter with FzfLua it uses ' instead of ` which jumps to the correct line, but not the correct column. 

By just changing the ' to ` in the goto_mark-function it jumps correctly:

```
M.goto_mark = function(selected)
...
  vim.cmd("normal! '" .. mark)
...
```

```
M.goto_mark = function(selected)
...
  vim.cmd("normal! `" .. mark)
...
```

I suggest both options should be available (maybe named "marks" and "marks_line")?

**Comment by damianoognissanti** [comment]
I created a suggestion and made a pull request. If backwards-compatibility is important the line+column mode could be the one with a new name, so that marks just takes you to the start of the line.

**Comment by ibhagwan** [comment]
Closed via #892, ty @damianoognissanti!
